docs/layouts/shortcodes/generated/deployment_configuration.html

```diff
@@ -12,7 +12,7 @@
             <td><h5>execution.allow-client-job-configurations</h5></td>
             <td style="word-wrap: break-word;">true</td>
             <td>Boolean</td>
-            <td>Determines whether configurations in the user program are allowed when running with Application mode. Has no effect for other deployment modes.</td>
+            <td>Determines whether configurations in the user program are allowed. Depending on your deployment mode failing the job might have different affects. Either your client that is trying to submit the job to an external cluster (session  cluster deployment) throws the exception or the Job manager (application mode deployment).</td>
```

Comment:
I think there bust be an extra space here as well between `session` and `cluster`